app/config/services.yml

```diff
@@ -191,6 +191,7 @@ services:
     Graby\Graby:
         arguments:
             $config:
+                http_client: { "ua_browser": "Wallabag/%wallabag.version% (Guzzle/5 +%domain_name%)" }
```

Comment:
I think that's a good idea because "webmaster" will be able to filter request from wallabag but I'm wondering how this is going to behave more globally because most websites might block request because they didn't know that _browser_.

The default UA is more likely to be accepted because it looks like a browser.

I'm wondering if this shouldn't be defined for each site which can create a dedicated rule to allow wallabag to request their content.

Reply:
Agreed, it can go both ways.

We are using Cloudflare and it sends JS challenges to Wallabag requests with the current default UA.
If someone didn't send me a post on Mastodon about this issue, I wouldn't have noticed that Wallabag traffic was blocked.

I think we can also argue that it's more fair to declare Wallabag instead of trying to hide it behind a browser.